lib/cocoapods/installer/sandbox_dir_cleaner.rb

```diff
@@ -62,6 +62,12 @@ def clean!
 
           removed_sandbox_private_headers = sandbox_private_headers(pod_targets) - sandbox_private_headers_to_install
           removed_sandbox_private_headers.each { |path| remove_dir(path) }
+
+          project_dir_names_to_install = pod_targets.map { |pod_target| sandbox.root + "#{pod_target.project_name}.xcodeproj" }
```

Comment:
should this be `sandbox.pod_target_project_path(pod_target.name)`?

Reply:
ah yeah nice.

Reply:
I would still use `sandbox.pod_target_project_path(pod_target.project_name)` though.